api/v1alpha1/rule_types_test.go

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
Are these auto-generated?

If not, would it be possible to write them with stretchr/testify and go tests / go subtests? We're using that style everywhere and it would be cool if we could keep consistency.